spec/requests/restaurants_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+RSpec.describe 'Restaurants', type: :request do
+  before(:all) do
+    FactoryBot.create_list(:restaurant, 10)
+  end
+
+  describe 'GET /restaurants' do
+    before do
+      get restaurants_path
+    end
+
+    it 'return https success' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'has the right objects count' do
+      arr = response_body
+      expect(arr.count).to eq(Restaurant.all.count)
+    end
+
+    it 'contains mandatory attributes' do
+      arr = response_body
+      arr.each do |json_rest|
+        expect(json_rest.keys).to include('id', 'name', 'cuisine', 'accept10bis')
+      end
+    end
+  end
+
+  describe 'GET /restaurant/:id' do
+    subject { Restaurant.all.sample }
```

Comment:
new line after + I would add a `context` for valid record and another one for the invalid record (and the subject will be in the context)
(context and describe are the same: http://lmws.net/describe-vs-context-in-rspec)